Contents/index.css

```diff
@@ -1,1514 +1,450 @@
-html,
```

Comment:
- [ ] Kindly as requested by the previous reviewer consider using the same font family as the one used in Figma design to make your work look as expected